docker-hub/access-tokens.md

```diff
@@ -28,15 +28,24 @@ Settings**.
 
 3. Select **Security > New Access Token**.
 
-      ![](images/hub-create-token.png)
-
 4. Add a description for your token. Use something that indicates where
 the token is going to be used, or set a purpose for the token.
 
-5. Copy the token that appears on the screen. Make sure you do this now:
+      ![PAT Menu](images/hub-create-token.png)
+
+5. You can view the following permissions from the drop-down:
+ * Read, Write, Delete: lets you to manage team members. This is an admin privilege.
+ * Read & Write: lets you to push images to any repository managed by your account.
+ * Read Only: lets you to view, search, and pull images from any public repositories and any private repositories that you have access to.
+ * Public Repo Read-only: lets you view, search, and pull images from any public repositories.
+
+ **Note**:
+ All users have access to Read, Write, and Delete. Only paid users have access to Read & Write, Read-only, and Public Repo Read-only.
+
+6. Copy the token that appears on the screen. Make sure you do this now:
 once you close this prompt, Docker will never show the token again.
 
-      ![](images/hub-copy-token.png)
+      ![Copy Token](images/hub-copy-token.png)
 
       Treat access tokens like your password and keep them secret. Store your tokens securely (for example, in a credential manager).
```

Comment:
Not new in this PR, but this warning doesn't stand out very well. I think this is important information, and perhaps should be mentioned at the top of the page. Perhaps we should move it to the introduction / start of the page and make it an admonition (warning)